5-Array-Union/Array Union.py

```diff
@@ -0,0 +1,54 @@
+def array_union(first, second):
+    result = []
+    try:
+        if first != ['']:
+            first = contains_2(first)
+            for number in first:
+                cond = contains(number, second)
+                if cond is True:
+                    delete_element(first, number)
+                elif cond is False:
+                    number = int(number)
+                    result.append(number)
+        if second != ['']:
+            second = contains_2(second)
+            for number in second:
+                number = int(number)
+                result.append(number)
+        return result
+    except ValueError:
+        if number in first:
+            first = delete_element(first, number)
+        elif number in second:
+            second = delete_element(second, number)
+        result = array_union(first, second)
+        return result
+
+
+def delete_element(array, element):
```

Comment:
Remove this function. It do nothing except ```array.remove(element)```. No reason to have a separate function for it.